--- composer-1.0.0-0.alpha8.3.ac497fe/vendor/composer/autoload_namespaces.php~	2014-06-20 16:57:55.000000000 +0300
+++ composer-1.0.0-0.alpha8.3.ac497fe/vendor/composer/autoload_namespaces.php	2014-06-20 16:58:38.314091739 +0300
@@ -4,8 +4,15 @@
 
 $vendorDir = dirname(dirname(__FILE__));
 $baseDir = dirname($vendorDir);
+$pearDir = PEAR_INSTALL_DIR . '/';
+$phpDir = defined('PHP_DATADIR') ? PHP_DATADIR . '/php/' : '';
 
 return array(
+    'Symfony\\Component\\Process\\' => array($pearDir),
+    'Symfony\\Component\\Finder' => array($pearDir),
+    'Symfony\\Component\\Console\\' => array($pearDir),
+    'Seld\\JsonLint' => array($phpDir),
+    'JsonSchema' => array($phpDir),
     'Composer\\Test' => array($baseDir . '/tests'),
     'Composer' => array($baseDir . '/src'),
 );
