--- composer-1.10.22/bin/composer~	2021-04-27 14:10:45.000000000 +0300
+++ composer-1.10.22/bin/composer	2021-05-10 18:32:01.839944783 +0300
@@ -6,7 +6,11 @@
 }
 
 setlocale(LC_ALL, 'C');
-require __DIR__.'/../src/bootstrap.php';
+if (file_exists($autoload = __DIR__ . '/../src/Composer/autoload.php')) {
+	require $autoload;
+} else {
+	require '/usr/share/php/Composer/autoload.php';
+}
 
 use Composer\Console\Application;
 use Composer\XdebugHandler\XdebugHandler;
--- composer-1.9.0/src/Composer/Json/JsonFile.php~	2019-08-02 21:55:33.000000000 +0300
+++ composer-1.9.0/src/Composer/Json/JsonFile.php	2019-08-11 19:59:58.343540127 +0300
@@ -34,7 +34,7 @@
     const JSON_PRETTY_PRINT = 128;
     const JSON_UNESCAPED_UNICODE = 256;
 
-    const COMPOSER_SCHEMA_PATH = '/../../../res/composer-schema.json';
+    const COMPOSER_SCHEMA_PATH = '/../res/composer-schema.json';
 
     private $path;
     private $rfs;
--- composer-1.10.22/src/Composer/Autoload/AutoloadGenerator.php~	2021-04-27 14:10:45.000000000 +0300
+++ composer-1.10.22/src/Composer/Autoload/AutoloadGenerator.php	2021-05-10 18:34:23.023946419 +0300
@@ -315,7 +315,7 @@
         $this->filePutContentsIfModified($targetDir.'/autoload_real.php', $this->getAutoloadRealFile(true, (bool) $includePathFileContents, $targetDirLoader, (bool) $includeFilesFileContents, $vendorPathCode, $appBaseDirCode, $suffix, $useGlobalIncludePath, $prependAutoloader, $staticPhpVersion));
 
         $this->safeCopy(__DIR__.'/ClassLoader.php', $targetDir.'/ClassLoader.php');
-        $this->safeCopy(__DIR__.'/../../../LICENSE', $targetDir.'/LICENSE');
+        $this->safeCopy(__DIR__.'/../res/LICENSE', $targetDir.'/LICENSE');
 
         if ($this->runScripts) {
             $this->eventDispatcher->dispatchScript(ScriptEvents::POST_AUTOLOAD_DUMP, $this->devMode, array(), array(
