--- ./src/Composer/Compiler.php.rpm	2016-04-18 22:14:28.000000000 +0200
+++ ./src/Composer/Compiler.php	2016-04-19 07:15:05.421424090 +0200
@@ -138,7 +138,7 @@ class Compiler
         }
         $this->addFile($phar, new \SplFileInfo(__DIR__.'/../../vendor/composer/ClassLoader.php'));
 
-        $this->addFile($phar, new \SplFileInfo(__DIR__ . '/../../res/cacert.pem'), false);
+        $this->addFile($phar, new \SplFileInfo('/etc/pki/tls/certs/ca-bundle.crt'), false);
 
         $this->addComposerBin($phar);
 
--- ./src/Composer/Util/RemoteFilesystem.php.rpm	2016-04-18 22:14:28.000000000 +0200
+++ ./src/Composer/Util/RemoteFilesystem.php	2016-04-19 07:15:05.421424090 +0200
@@ -925,7 +925,7 @@ class RemoteFilesystem
             }
         }
 
-        return $caPath = __DIR__.'/../../../res/cacert.pem'; // Bundled with Composer, last resort
+        return $caPath = '/etc/pki/tls/certs/ca-bundle.crt';
     }
 
     /**
