--- composer-1.0.0-12.alpha10.216.g5744981/src/Composer/Compiler.php~	2015-08-12 17:36:25.000000000 +0300
+++ composer-1.0.0-12.alpha10.216.g5744981/src/Composer/Compiler.php	2015-08-12 17:37:00.837855396 +0300
@@ -96,24 +96,6 @@
         foreach ($finder as $file) {
             $this->addFile($phar, $file, false);
         }
-        $this->addFile($phar, new \SplFileInfo(__DIR__ . '/../../src/Composer/IO/hiddeninput.exe'), false);
-
-        $finder = new Finder();
-        $finder->files()
-            ->ignoreVCS(true)
-            ->name('*.php')
-            ->name('LICENSE')
-            ->exclude('Tests')
-            ->exclude('tests')
-            ->exclude('docs')
-            ->in(__DIR__.'/../../vendor/symfony/')
-            ->in(__DIR__.'/../../vendor/seld/jsonlint/')
-            ->in(__DIR__.'/../../vendor/justinrainbow/json-schema/')
-        ;
-
-        foreach ($finder as $file) {
-            $this->addFile($phar, $file);
-        }
 
         $this->addFile($phar, new \SplFileInfo(__DIR__.'/../../vendor/autoload.php'));
         $this->addFile($phar, new \SplFileInfo(__DIR__.'/../../vendor/composer/autoload_namespaces.php'));
--- composer-1.0.0-12.alpha10.216.g5744981/composer.json~	2015-04-15 00:18:51.000000000 +0300
+++ composer-1.0.0-12.alpha10.216.g5744981/composer.json	2015-08-12 17:37:36.376332853 +0300
@@ -22,12 +22,7 @@
         "issues": "https://github.com/composer/composer/issues"
     },
     "require": {
-        "php": ">=5.3.2",
-        "justinrainbow/json-schema": "~1.4",
-        "seld/jsonlint": "~1.0",
-        "symfony/console": "~2.5",
-        "symfony/finder": "~2.2",
-        "symfony/process": "~2.1"
+        "php": ">=5.3.2"
     },
     "require-dev": {
         "phpunit/phpunit": "~4.5"
