--- composer-1.0.x-b23a3cd/src/Composer/Compiler.php~	2014-11-28 15:47:34.000000000 +0200
+++ composer-1.0.x-b23a3cd/src/Composer/Compiler.php	2014-11-28 15:53:21.829773210 +0200
@@ -40,13 +40,13 @@
             unlink($pharFile);
         }
 
-        $process = new Process('git log --pretty="%H" -n1 HEAD', __DIR__);
+        $process = new Process('echo $PACKAGE_VERSION', __DIR__);
         if ($process->run() != 0) {
             throw new \RuntimeException('Can\'t run git log. You must ensure to run compile from composer git repository clone and that git binary is available.');
         }
         $this->version = trim($process->getOutput());
 
-        $process = new Process('git log -n1 --pretty=%ci HEAD', __DIR__);
+        $process = new Process('echo $RELEASE_DATE', __DIR__);
         if ($process->run() != 0) {
             throw new \RuntimeException('Can\'t run git log. You must ensure to run compile from composer git repository clone and that git binary is available.');
         }
@@ -55,7 +55,7 @@
         $date->setTimezone(new \DateTimeZone('UTC'));
         $this->versionDate = $date->format('Y-m-d H:i:s');
 
-        $process = new Process('git describe --tags HEAD');
+        $process = new Process('false');
         if ($process->run() == 0) {
             $this->version = trim($process->getOutput());
         } else {
