--- composer-1.0.x-5744981/src/Composer/Compiler.php~	2015-02-18 00:33:55.000000000 +0200
+++ composer-1.0.x-5744981/src/Composer/Compiler.php	2015-02-18 00:34:38.952650516 +0200
@@ -40,13 +40,13 @@
             unlink($pharFile);
         }
 
-        $process = new Process('git log --pretty="%H" -n1 HEAD', __DIR__);
+        $process = new Process('echo $PACKAGE_VERSION', __DIR__);
         if ($process->run() != 0) {
             throw new \RuntimeException('Can\'t run git log. You must ensure to run compile from composer git repository clone and that git binary is available.');
         }
         $this->version = trim($process->getOutput());
 
-        $process = new Process('git log -n1 --pretty=%ci HEAD', __DIR__);
+        $process = new Process('echo $RELEASE_DATE', __DIR__);
         if ($process->run() != 0) {
             throw new \RuntimeException('Can\'t run git log. You must ensure to run compile from composer git repository clone and that git binary is available.');
         }
@@ -55,7 +55,7 @@
         $date->setTimezone(new \DateTimeZone('UTC'));
         $this->versionDate = $date->format('Y-m-d H:i:s');
 
-        $process = new Process('git describe --tags --exact-match HEAD');
+        $process = new Process('echo $PACKAGE_VERSION; exit $DEV_VERSION');
         if ($process->run() == 0) {
             $this->version = trim($process->getOutput());
         } else {
--- composer-1.0.0-14.alpha11/bin/compile	2015-11-25 18:32:33.718541181 +0200
+++ composer-1.0.0-14.alpha11/bin/compile	2015-11-25 19:04:02.596489328 +0200
@@ -1,20 +1,6 @@
 #!/usr/bin/php
 <?php
 
-$cwd = getcwd();
-chdir(__DIR__.'/../');
-$ts = rtrim(shell_exec('git log -n1 --pretty=%ct HEAD'));
-if (!is_numeric($ts)) {
-    echo 'Could not detect date using "git log -n1 --pretty=%ct HEAD"'.PHP_EOL;
-    exit(1);
-}
-// Install with the current version to force it having the right ClassLoader version
-// Install without dev packages to clean up the included classmap from phpunit classes
-shell_exec('php bin/composer config autoloader-suffix ComposerPhar' . $ts);
-shell_exec('php bin/composer install -q --no-dev');
-shell_exec('php bin/composer config autoloader-suffix --unset');
-chdir($cwd);
-
 require __DIR__.'/../src/bootstrap.php';
 
 use Composer\Compiler;
