--- composer-alpha7/src/Composer/Compiler.php~	2013-04-01 13:18:27.000000000 +0300
+++ composer-alpha7/src/Composer/Compiler.php	2013-04-01 22:27:27.313555304 +0300
@@ -35,6 +35,8 @@
             unlink($pharFile);
         }
 
+		$this->version = getenv('COMPOSER_VERSION');
+	if (empty($this->version)) {
         $process = new Process('git log --pretty="%H" -n1 HEAD', __DIR__);
         if ($process->run() != 0) {
             throw new \RuntimeException('Can\'t run git log. You must ensure to run compile from composer git repository clone and that git binary is available.');
@@ -45,6 +47,7 @@
         if ($process->run() == 0) {
             $this->version = trim($process->getOutput());
         }
+	}
 
         $phar = new \Phar($pharFile, 0, 'composer.phar');
         $phar->setSignatureAlgorithm(\Phar::SHA1);
