--- composer-1.3.0/src/Composer/Downloader/SvnDownloader.php~	2016-12-24 01:47:04.000000000 +0200
+++ composer-1.3.0/src/Composer/Downloader/SvnDownloader.php	2016-12-25 00:20:44.725141875 +0200
@@ -41,7 +41,7 @@
         }
 
         $this->io->writeError(" Checking out ".$package->getSourceReference());
-        $this->execute($url, "svn co", sprintf("%s/%s", $url, $ref), null, $path);
+        $this->execute($url, "svn co --ignore-externals", sprintf("%s/%s", $url, $ref), null, $path);
     }
 
     /**
@@ -64,7 +64,7 @@
         }
 
         $this->io->writeError(" Checking out " . $ref);
-        $this->execute($url, "svn switch" . $flags, sprintf("%s/%s", $url, $ref), $path);
+        $this->execute($url, "svn switch --ignore-externals" . $flags, sprintf("%s/%s", $url, $ref), $path);
     }
 
     /**
