--- composer-1.0.0-13.alpha10/src/Composer/Downloader/SvnDownloader.php~	2015-04-15 00:18:51.000000000 +0300
+++ composer-1.0.0-13.alpha10/src/Composer/Downloader/SvnDownloader.php	2015-09-02 21:37:14.936187207 +0300
@@ -156,7 +156,7 @@
             $fromRevision = preg_replace('{.*@(\d+)$}', '$1', $fromReference);
             $toRevision = preg_replace('{.*@(\d+)$}', '$1', $toReference);
 
-            $command = sprintf('svn log -r%s:%s --incremental', $fromRevision, $toRevision);
+            $command = sprintf('svn log -r%s:%s --incremental --ignore-externals', $fromRevision, $toRevision);
 
             if (0 !== $this->process->execute($command, $output, $path)) {
                 throw new \RuntimeException(
