--- composer-1.0.0/vendor/composer/autoload_namespaces.php~	2013-04-12 18:41:14.000000000 +0300
+++ composer-1.0.0/vendor/composer/autoload_namespaces.php	2013-04-12 18:42:10.735493349 +0300
@@ -4,8 +4,12 @@
 
 $vendorDir = dirname(dirname(__FILE__));
 $baseDir = dirname($vendorDir);
+$pearDir = PEAR_INSTALL_DIR . '/';
 
 return array(
+    'Symfony\\Component\\Process\\' => $pearDir,
+    'Symfony\\Component\\Finder' => $pearDir,
+    'Symfony\\Component\\Console\\' => $pearDir,
     'Seld\\JsonLint' => $vendorDir . '/seld/jsonlint/src/',
     'JsonSchema' => $vendorDir . '/justinrainbow/json-schema/src/',
     'Composer' => $baseDir . '/src/',
